debian/changelog

```diff
@@ -8,9 +8,9 @@ cloud-init (23.1-0ubuntu0~18.04.3) UNRELEASED; urgency=medium
     - d/p/openstack-no-network-patch.patch
     - d/p/netplan99-cannot-use-default.patch
     - d/p/retain-netplan-world-readable.patch
-  * Upstream snapshot based on upstream/main at 4b6e4e13.
```

Comment:
Since this previous snapshot wasn't released. Let's just manually redact this line

Reply:
I already got rid of the old one. Are you saying I should get rid of the new one too?